content/source/docs/enterprise-beta/saml/index.html.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: enterprise2
+page_title: "SAML - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-saml"
+---
+
+# SAML Single Sign On
+
+-> **Note**: The SAML Single Sign On feature is only available with the Premium tier on private installs.
+
+SAML is an XML-based standard for authentication and authorization. Terraform Enterprise can act as a service provider (SP) (or Relying Party) with your internal SAML identity provider (IdP).
+
+Terraform Enterprise supports the SAML 2.0 standard. It has been tested with a variety of identity providers.
+
+- Azure Active Directory (Azure AD) - planned
```

Comment:
Does "planned" mean it doesn't currently work, or just that we haven't tested whether it works? As a user of one of these, I want that to be more explicit.